pyproject.toml

```diff
@@ -15,16 +16,21 @@ classifiers = [
     "License :: OSI Approved :: Apache Software License",
     "Operating System :: OS Independent",
 ]
+dependencies = [
+    "flit",
```

Comment:
I added this because you used to have this in the INSTALL file:
```
## 4. Install dependencies

The flit build system is used to install dependencies. Make sure you have flit installed:


pip install flit


To install the base dependencies to run the code:
```

so instead of an extra install step I added it to the dependencies.